src/Command/RefundUnits.php

```diff
@@ -27,8 +28,8 @@ final class RefundUnits
 
     public function __construct(string $orderNumber, array $units, array $shipments, int $paymentMethodId, string $comment)
     {
-        Assert::allIsInstanceOf($units, OrderItemUnitRefund::class);
-        Assert::allIsInstanceOf($shipments, ShipmentRefund::class);
+        Assert::allIsInstanceOf($units, UnitRefundInterface::class);
+        Assert::allIsInstanceOf($shipments, UnitRefundInterface::class);
```

Comment:
Typehints of variables `$units` and `$shipments` and getter methods should be also adjusted

Reply:
What do you mean?